models/org.go

```
@@ -172,7 +172,9 @@ func CreateOrganization(org, owner *User) (err error) {
 	}
 
 	if _, err = sess.Insert(&units); err != nil {
-		sess.Rollback()
+		if err := sess.Rollback(); err != nil {
+			return err
```

IMHO this error should only be logged and the method should return the initial error returned by the `Insert()`.
Same for L488 and L493

---

Sounds like a good idea, thanks.